tests/mpt/data.py

```diff
@@ -0,0 +1,52 @@
+NOTIFY_GET_RESP = {
```

Comment:
Usually we are using fixtures for it, and probably you can improve fixture like

```
@pytest.fixture()
def mpt_list_response():
    def _wrap_response(objects_list):
        return {
            "data": objects_list,
        }

    return _wrap_response
```

to add paginated information for you here and reuse it in tests